### PR TITLE
documentation: Installing dev environment  Fix #725

### LINKED ISSUE #725: documentation for Installing dev environment
Motivation
----------

documentation for Installing dev environment

### PR DESCRIPTION
**Pull request by vingar**

### CODE REVIEW

**Comment by mlassnig** [comment]
while True:
    increase_commit_count() 

:-)

**Comment by vingar** [comment]
on purpose as it's a constant fight to get travis working on squashed commits

**Comment by bari12** [comment]
Ideally you should just make a PR for something which is review-ready and not every development iteration.

File: etc/docker/dev/README.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+   $> docker ps
+   CONTAINER ID        IMAGE                    COMMAND                  CREATED             STATUS                    PORTS                  NAMES
+   3d9dfb317ada        rucio-dev                "httpd -D FOREGROUND"    28 minutes ago      Up 28 minutes             0.0.0.0:443->443/tcp   dev_rucio_1
+   88a5ad6cf6d0        mysql/mysql-server:5.7   "/entrypoint.sh my..."   28 minutes ago      Up 28 minutes (healthy)   3306/tcp, 33060/tcp    dev_mysql_1
+
+
+Note that the names of the containers may end up looking different on your system. To run the test suite use `docker exec` with the rucio container, e.g.::
+
+   $> docker exec -it dev_rucio_1 tools/run_tests.sh
+
```
Comment (mlassnig) [inline etc/docker/dev/README.rst:66]: Add section on how to enter the docker container?

Comment (vingar) [reply etc/docker/dev/README.rst:66]: point added.

**Comment by vingar** [comment]
I didn't notice the incompatibility between the markup language and the Restructured Text (reST) one at glance. Each README, etc should be written in RST for further integration in the documentation.

commits squashed and Travis looks happy

File: etc/docker/dev/README.md [hunk 2]
```diff
@@ -1,45 +0,0 @@
...
-3d9dfb317ada        rucio-dev                "httpd -D FOREGROUND"    28 minutes ago      Up 28 minutes             0.0.0.0:443->443/tcp   dev_rucio_1
-88a5ad6cf6d0        mysql/mysql-server:5.7   "/entrypoint.sh my..."   28 minutes ago      Up 28 minutes (healthy)   3306/tcp, 33060/tcp    dev_mysql_1
-```
-Note that the names of the containers may end up looking different on your system. To run the test suite use `docker exec` with the rucio container, e.g.
-```
-docker exec -it dev_rucio_1 tools/run_tests.sh
-```
-
-### Git Hook
-Replace the `pre-commit` hook provided in `tools` with the one provided here. It runs `pylint` in the rucio development container rather then in the local system.
```
Comment (bari12) [inline etc/docker/dev/README.md:45]: The git hook we should rather mention in the CONTRIBUTING guide I feel. (We have the tools/configure_git.sh script for this as well)

Comment (vingar) [reply etc/docker/dev/README.md:45]: I updated both files but there is a difference in the second pre-commit hook

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -116,3 +116,11 @@ Coding Style and testing
 
 We use flake8 and pylint to sanitize our code. Please do the same before
 submitting a pull request.
+
+
+Git Hooks
+---------
+
+Some git hooks (pre-commit, prepare-commit-msg) can installed by executing the script tools/configure_git.sh::
```
Comment (bari12) [inline CONTRIBUTING.rst]: can be installed

**Review by mlassnig — APPROVED** [review]

**Pull request merged**